flight/Modules/Actuator/actuator.c

```diff
@@ -343,29 +370,53 @@ static void actuator_task(void* parameters)
 			clipped = true;
 
 			offset = 1.0f - max_chan;
+		} else if (min_chan < 0.0f) {
+			clipped = true;
+
+			/* Low-side clip management-- how much power are we
+			 * willing to add??? */
+
+			neg_clip /= num_motors;
```

Comment:
I wonder if using the mean is the right thing to do.. wouldn't using the maximum neg clip be better?